models/Tabs.model.js

```diff
@@ -2,22 +2,11 @@ const { Schema, model } = require("mongoose");
 
 const tabSchema = new Schema(
   {
-    category: {
-      enum: ["video", "article", "podcast"],
-      required: [true, "Select a category: video, article, podcast"],
-      trim: true,
-    },
-    title: {
-      type: String,
-      required: [true, "Add a short title"],
-      trim: true,
-    },
+    category: String,
+    title: String,
```

Comment:
A few of these I would assume would be required? Or why not?

Reply:
hmm the category could be required, but as the rest is coming from the LinkPreview API and not all links will have images etc I didnt want to make it compulsory.